Packs/MalwareBazaar/Integrations/MalwareBazaar/MalwareBazaar.yml

```
@@ -0,0 +1,446 @@
+      description: Dynamic malware analysis from ANY.RUN.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.CAPE
+      description: Dynamic malware analysis from CAPE Sandbox.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.CERT-PL_MWDB
+      description: Threat intel from CERT.PL Malware Database.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.vxCube
+      description: Dynamic malware analysis from Dr.Web vxCube.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.DocGuard
+      description: Office document reputation from DocGuad.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.FileScan-IO
+      description: Malware analysis service from FileScan.IO.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.InQuest Labs
+      description: File reputation service from InQuest Labs.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.Intezer
+      description: Code analysis from Intezer.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.ReversingLabs
+      description: File reputation & intelligence from ReversingLabs TitaniumCloud.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.Spamhaus_HBL
+      description: File reputation from Spamhaus Hash Blocklist (HBL).
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.Triage
+      description: Dynamic malware analysis from Hatching Triage.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.UnpacMe
+      description: Malware unpacking service from UnpacMe.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.VMRay
+      description: Dynamic malware analysis from VMRay.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.vendor_intel.YOROI_YOMI
+      description: Dynamic malware analysis from YOROI YOMI.
+      type: Unknown
+    - contextPath: MalwareBazaar.File.comments.id
+      description: Unique id that identifies this comment.
+      type: String
+    - contextPath: MalwareBazaar.File.comments.date_added
+      description: Timestamp (UTC) when this comment has been made.
+      type: Date
+    - contextPath: MalwareBazaar.File.comments.twitter_handle
+      description: Twitter handle who wrote this comment.
+      type: String
+    - contextPath: MalwareBazaar.File.comments.display_name
+      description: Twitter display name.
+      type: String
+    - contextPath: MalwareBazaar.File.comments.comment
+      description: The comment itself.
+      type: String
+  - arguments:
+    - default: false
+      description: SHA256 hash of the malware sample to download.
+      isArray: false
+      name: sha256_hash
+      required: true
+      secret: false
+    deprecated: false
+    description: Download (fetch) malware samples from MalwareBazaar. Any malware
```

```suggestion
    description: Download malware samples from MalwareBazaar. Any malware
```
fetch is a "reserved word" in XSOAR, we use it only in fetch incidents/indicators